docs/hub/post_install.md

```diff
@@ -27,145 +26,13 @@ lastupdated: "2021-02-20"
-    cd ibm-eam-{{site.data.keyword.semver}}-agent-x86_64/tools
-    ```
-    {: codeblock}
-5. Validate the installation state:
-    ```
-    ./service_healthcheck.sh
-    ```
-    {: codeblock}
-
-    See the following example output:
-    ```
-    $ ./service_healthcheck.sh
-    ==Running service verification tests for IBM Edge Application Manager==
-    SUCCESS: IBM Edge Application Manager Exchange API is operational
-    SUCCESS: IBM Edge Application Manager Cloud Sync Service is operational
-    SUCCESS: IBM Edge Application Manager Agbot database heartbeat is current
-    SUCCESS: IBM Edge Application Manager SDO API is operational
-    SUCCESS: IBM Edge Application Manager UI is properly requiring valid authentication
-    ==All expected services are up and running==
-    ```
-
-   * If there are **service_healthcheck.sh** command failures, if you experience issues running the commands below, or if there are issues during runtime, see [troubleshooting](../admin/troubleshooting.md).
 
 
 ## Post installation configuration
 {: #postconfig}
 
 The following process must run on a host that supports installation of the **hzn** CLI, which currently can be installed on a Debian/apt based Linux, amd64 Red Hat/rpm Linux, or macOS host. These steps use the same media downloaded from PPA in the Installation verification section.
 
-1. Install the **hzn** CLI using the instructions for your supported platform:
-  * Navigate to the **agent** directory and unpack the agent files:
-    ```
-    cd ibm-eam-{{site.data.keyword.semver}}-agent-x86_64/agent && \
-    tar -zxvf edge-packages-{{site.data.keyword.semver}}.tar.gz
-    ```
-    {: codeblock}
-
-    * Debian {{site.data.keyword.linux_notm}} example:
-      ```
-      sudo apt-get install ./edge-packages-{{site.data.keyword.semver}}/linux/deb/amd64/horizon-cli*.deb
-      ```
-      {: codeblock}
-
-    * Red Hat {{site.data.keyword.linux_notm}} example:
-      ```
-      sudo dnf install -yq ./edge-packages-{{site.data.keyword.semver}}/linux/rpm/x86_64/horizon-cli-*.x86_64.rpm
-      ```
-      {: codeblock}
-
-    * macOS example:
-      ```
-      sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain edge-packages-{{site.data.keyword.semver}}/macos/pkg/x86_64/horizon-cli.crt && \
-      sudo installer -pkg edge-packages-{{site.data.keyword.semver}}/macos/pkg/x86_64/horizon-cli-*.pkg -target /
-      ```
-      {: codeblock}
-
-2. Run the post installation script. This script performs all the necessary initialization to create your first organization. (Organizations are how {{site.data.keyword.ieam}} separates resources and users to enable multi-tenancy. Initially, this first organization is sufficient. You can configure more organizations later. For more information, see [Multi-tenancy ![Opens in a new tab](../images/icons/launch-glyph.svg "Opens in a new tab")](../admin/multi_tenancy.md)).
```

Comment:
I would take the sentences out of the ( ). Just make them "reg" sentences in the step.